### PR TITLE
fix: Oracle price feeders add verification of empty subscribed pairs

### PR DESCRIPTION
**Pull request by RafilxTenfen**
## Description

- Add verification and tests for `SubscribedPairs` with empty parameters

closes: #672 

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added appropriate labels to the PR
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by adamewozniak — APPROVED** [review]
🚀

File: price-feeder/CHANGELOG.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Bug Fixes
 
 - [#607](https://github.com/umee-network/umee/pull/607) Fix kraken provider timestamp unit.
+- [#675](https://github.com/umee-network/umee/pull/675) Add verification for subscribe currency pairs with empty parameter.
```
[the remark this answers is not in the document]
Comment (RafilxTenfen) [reply price-feeder/CHANGELOG.md]: done at https://github.com/umee-network/umee/pull/675/commits/ad149e6a6645d8f33ca4b4269c7fa4390a013314

File: price-feeder/CHANGELOG.md [hunk 2]
```diff
@@ -49,6 +49,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 - [#648](https://github.com/umee-network/umee/pull/648) Add Coinbase as a provider.
 
+### Bug Fixes
+
+- [#675](https://github.com/umee-network/umee/pull/675) Add verification for subscribe currency pairs with empty parameter.
```
Comment (adamewozniak) [inline price-feeder/CHANGELOG.md:54]: Forgot to mention, I would also update this to say : 
```
- [#675](https://github.com/umee-network/umee/pull/675) Add necessary input validation to SubscribePairs in the price feeder.
```

Comment (RafilxTenfen) [reply price-feeder/CHANGELOG.md:54]: agreed, thanks!

Comment (RafilxTenfen) [reply price-feeder/CHANGELOG.md]: done at https://github.com/umee-network/umee/pull/675/commits/84374f0388352ba1db1bb6ae95cca967d0604ff0
